contracts/BSCValidatorSet.sol

```diff
@@ -366,6 +399,10 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
       }
     }
     // averageDistribute*rest may less than income, but it is ok, the dust income will go to system reward eventually.
+
+    if (can) {
```

Comment:
can we refactor this `can`, it's far away from where it's defined

Reply:
removed code